contracts/bond/Redeemable.sol

```diff
@@ -41,6 +43,10 @@ abstract contract Redeemable is Initializable {
         return _redeemable;
     }
 
+    function reason() external view returns (string memory) {
```

Comment:
suggestion: rename to `redemptionReason`.

With the propagation of reason messages, if we have a number of abstract contracts, it'll be easier if the getters are named uniquely.